lib/much-rails/action/save.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require "much-rails/action"
+require "much-rails/change_action_result"
+require "much-rails/config"
+require "much-rails/plugin"
+
+# MuchRails::Action::Save defines the common behaviors for all view
+# action classes that save records.
+module MuchRails; end
+module MuchRails::Action; end
+module MuchRails::Action::Save
```

Comment:
Name is super weird. I'm thinking this should be renamed to `lib/much-rails/save_action.rb` and `MuchRails::SaveAction`.

Thoughts?

Same goes for coming `MuchRails::{Remove|Destroy}Action`s